index.bs

```diff
@@ -271,17 +271,16 @@ dictionary CookieStoreDeleteOptions {
   USVString? domain = null;
   USVString path = "/";
   boolean secure = true;
-  boolean httpOnly = false;
   CookieSameSite sameSite = "strict";
 };
 
 dictionary CookieListItem {
   USVString name;
```

Comment:
Good point re: the CookieChangeEvent constructor. 

Should `name`/`value` be required then?

Reply:
Done. Thanks for the catch!

Reply:
I spoke too soon. `value` cannot be required, because it's missing for deleted cookie info. I updated the PR to reflect this.

We could do an empty value instead of missing value key, if you think that'd more usable, but that's a separate discussion.